metricflow/api/metricflow_client.py

```diff
@@ -27,9 +28,15 @@ class MetricFlowClient:
     """MetricFlow Python client for running basic queries and other standard commands."""
 
     @staticmethod
-    def from_config() -> MetricFlowClient:
-        """Builds a MetricFlowClient via config yaml file stored in the designated config location."""
-        handler = ConfigHandler()
+    def from_config(config_file_path: Optional[str] = None) -> MetricFlowClient:
+        """Builds a MetricFlowClient via config yaml file.
+
+        If config_file_path is not passed, it will use the default config location.
+        """
+        if config_file_path is not None:
+            handler = YamlFileHandler(yaml_file_path=config_file_path)
```

Comment:
Nice. We might want to add a logger.debug() or something here to allow people to track which config path they're using when overrides are present. Or maybe just always do a logger.debug on the config file path, either way. Up to you if you want to add that or not.